packages/types/lib/runner/index.ts

```diff
@@ -1,12 +1,14 @@
 export interface RunnerOutputError {
     type: string;
-    payload: Record<string, unknown>;
+    payload: Record<string, unknown> | unknown[];
```

Comment:
where can payload be an array?

Reply:
This was a super weird one once I started cleaning up types in `exec.ts`. 

See https://github.com/NangoHQ/nango/blob/b61728bf82e27631eb9ec73fd9b6e771ec9f38dd/packages/runner/lib/exec.ts#L117. `valInput` is a `ValidateDataError[]`. I'm pretty stumped as to why typescript wasn't picking that up as an error before these changes, though.